TrueCraft.Core/Logic/Items/BookItem.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using TrueCraft.API.Logic;
+
+namespace TrueCraft.Core.Logic.Items
+{
+    public class BookItem : ItemProvider
+    {
+        public static readonly short ItemID = 0x154;
+
+        public override short ID { get { return 0x154; } }
+
+        public override sbyte MaximumStack { get { return 64; } }
```

Comment:
Don't override this unless it's unusual. The default is 64 and you can omit this if the item in question can be stacked to 64.